src/node/build/build.ts

```diff
@@ -65,8 +65,8 @@ export async function build(
 
       const pages = ['404.md', ...siteConfig.pages]
 
-      for (const page of pages) {
-        await renderPage(
+      await Promise.allSettled(
```

Comment:
Any specific reason to use `allSettled` instead of `all`? `allSettled` will always get resolved (even if there are errors while rendering).

Reply:
Sorry, I thought no need to throw an error even if some page failed to generate.

So do I need to change it to `Promise.all` ?

Reply:
Yeah change it to `Promise.all` and fix linting errors (run `pnpm run lint` before committing).

Reply:
> Yeah change it to `Promise.all` and fix linting errors (run `pnpm run lint` before committing).

Extremely sorry, I used github web based editor to commit the changes :( so, I dont have choice to run cli commands (my local machine having some issues) so can you please help me with it :) (sorry for incontinent )